src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -91,6 +92,16 @@
 
     private final EventBus eventBus = new EventBus();
 
+    private DatabaseLocation location;
+
+    public BibDatabase() {
+        this(DatabaseLocation.LOCAL);
+    }
+
+    public BibDatabase(DatabaseLocation location) {
```

Comment:
:+1: I didn't realize that, thans for the suggestion. This is going to be changed.